src/share-button.js

```diff
@@ -40,6 +40,10 @@ ShareButton.prototype.setContent = function (content) {
 };
 
 ShareButton.prototype._execute = function (evt) {
+    // prevent space from scrolling the page
+    if (evt.type === "keyup" && evt.which === 32) {
+        evt.preventDefault();
+    }
     if (evt.which !== 13 && evt.which !== 32 && evt.type === 'keyup') {
```

Comment:
Don't these key checks get run by the parent later, when Button.prototype._execute is called?